### PR TITLE
Bump Mainflux to v0.11.0

### PR DESCRIPTION
**Pull request by netoax**
What does this implement/fix? Explain your changes.
---------------------------------------------------
This patch bumps the Mainflux services to v0.11.0. In addition to that, some env variables names were updated to keep compatibility with this new version.

Does this close any currently open issues?
------------------------------------------

nops.

Any relevant logs, error output, etc?
-------------------------------------

Any other comments?
-------------------
nops.

Where has this been tested?
---------------------------

**Operating System/Platform:** macOS Darwin - 18.0.0

**NPM Version:** 6.14.4

**NodeJS Version:** v12.16.2

### CODE REVIEW

File: stacks/core/dev/env.d/knot-cloud-storage.env [hunk 1]
```diff
@@ -1,4 +1,4 @@
 DB_HOSTNAME=mongo
 DB_PORT=27017
 USERS_HOST=things
```
Comment (netoax) [inline stacks/core/dev/env.d/knot-cloud-storage.env]: @esdrasjnr  I would like to rename this to `THINGS_HOST` as I commented on your PR. So, maybe I could wait for it, right?

**Review by esdrasjnr — COMMENTED** [review]
I'm just wondering about the compatibility with code. I remember when I tried to update, the `users` and `things` services have been broken. They are running ok with these changes?

Anyway, it LGTM! But I'll wait for your answer to approve :+1: :100:

**Review by esdrasjnr — APPROVED** [review]
As you confirmed that everything is OK, approved! :smiley:

**Review by Igoraamc — APPROVED** [review]
LGTM! 🥳

**Pull request merged**